### ISSUE TITLE
The wgrep pop-up buffer cannot be selected when C-c C-e is performed on directory or project search result

### ISSUE THREAD

**Issue by ylorn** [hlissner/doom-emacs#5361]
### What did you expect to happen?

The writable pop-up buffer of the directory or project search result should be accessible to perform modification

### What actually happened?

The writable pop-up buffer of the directory or project search result is not selectable.

### Describe your attempts to resolve the issue

I have tried:

1. `SPC w j` to select the bottom pop-up window;
2. use the mouse to click on the bottom pop-up window;

Neither of them work to let me edit the writable buffer.

There is a workaround, if I press `SPC b i` and select the wgrep buffer explicitly, then the buffer will be opened in full window, and I can modify the buffer there without any problem. However, when I try to use the `Z Z` to apply the modification and quit the buffer, only the first part will work: the changes will be applied to each candidate file successfully, but the buffer won't be closed with prompt: 

> Attempt to delete minibuffer or sole ordinary window

This problem happens in both `ivy` and `vertico` for me.

However, the in-file search works fine. That is to say, if I perform the `C-c C-e` on the `SPC s s` result, then the pop-up window is automatically selected and can be modified and can be closed properly with `Z Z`.

So it seem it only happens for project and directory search result.

### Steps to reproduce

1. `SPC p p` to open some project;
2. `SPC s p` to search some query;
3. `C-c C-e` to export the search result to a wgrep buffer;
4. A pop-up window is brought up and cannot be selected.

### System Information

> Executing 'doom info' with Emacs 27.2 at 2021-08-09 23:46:29     generated    Aug 09, 2021 23:46:29       system       MacOS 11.5.1 Darwin 20.6.0 arm64       emacs        27.2 ~/.emacs.d/ -> ~/.emacs.d/       doom         3.0.0-alpha HEAD -> develop 6678f3833 2021-08-06 03:50:33 -0400 ~/.doom.d/ -> ~/.doom.d/       shell        /bin/zsh       features     RSVG DBUS GLIB NOTIFY KQUEUE ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS XIM MODULES THREADS JSON PDUMPER GMP       traits       batch server-running       modules      :completion company (vertico +icons) :ui doom doom-dashboard doom-quit hl-todo indent-guides minimap modeline ophints (popup +defaults) treemacs unicode vc-gutter vi-tilde-fringe workspaces :editor (evil +everywhere) file-templates fold multiple-cursors snippets :emacs (dired +icons) electric (ibuffer +icons) undo vc :term eshell vterm :checkers syntax (spell +everywhere) :tools debugger docker (eval +overlay) lookup lsp magit make pdf tmux upload :os macos (tty +osc) :lang (cc +lsp) emacs-lisp (json +lsp) javascript lua markdown (org +brain +dragndrop +journal +noter +pretty +roam) python (ruby +rails) sh (swift +lsp) yaml :email (mu4e +gmail) :app irc (rss +org) twitter :config (default +bindings +smartparens)       packages     (dmenu) (elfeed) (emms) (emojify) (evil-tutor) (mastodon) (md4rd) (peep-dired) (rainbow-mode) (sublimity) (tldr) (wc-mode) (writeroom-mode) (leetcode) (solaire-mode :disable t) (zoom)

**Comment by hlissner** [comment]
I can't reproduce this in vanilla Doom, so I suspect it's a private configuration issue. Please try to reproduce this [in vanilla Doom using Doom's sandbox](https://github.com/hlissner/doom-emacs/blob/develop/docs/getting_started.org#testing-in-dooms-sandbox).

**Comment by ylorn** [comment]
You are right. It is due to my private config. The following lines causes the problem:

```elisp
(require 'sublimity-scroll)
(require 'sublimity-map)
(require 'sublimity-attractive)
(sublimity-mode 0)
```

Sorry for the fuss.
